src/qmozview_p.cpp

```diff
@@ -1518,6 +1523,23 @@ bool QMozViewPrivate::handleAsyncMessage(const QString &message, const QVariant
         QInputMethod *inputContext = qGuiApp->inputMethod();
         inputContext->update(Qt::ImSurroundingText | Qt::ImCursorPosition | Qt::ImAnchorPosition);
         return true;
+    } else if (message == QLatin1String(INPUTMETHOD_SET_INPUT_ATTRIBUTES)) {
+        QVariantMap map = data.toMap();
+        mInputMethodAttributes = 0;
+        if (map.value(QLatin1String("autocomplete")) == "off" || map.value(QLatin1String("autocomplete")) == "none") {
```

Comment:
```suggestion
        if (map.value(QLatin1String("autocomplete")) == QLatin1String("off") || map.value(QLatin1String("autocomplete")) == QLatin1String("none")) {
```

Sorry missed this earlier... If line is too long, split it into two lines. Same for the other conditions.